lib/assets/resign.sh

```diff
@@ -92,65 +107,68 @@ NESTED_APP_REFERENCE_KEYS=(":WKCompanionAppBundleIdentifier" ":NSExtension:NSExt
 
 # options start index
 OPTIND=3
-while getopts p:d:e:k:b:n: opt; do
+while getopts p:d:e:k:b:n:v opt; do
     case $opt in
         p)
             RAW_PROVISIONS+=("$OPTARG")
-            if [[ "$OPTARG" =~ .+=.+ ]]; then
-                echo "Specified provisioning profile: '${OPTARG#*=}' for bundle identifier: '${OPTARG%%=*}'" >&2
-            else
-                echo "Specified provisioning profile: '$OPTARG'" >&2
-            fi
             ;;
         d)
             DISPLAY_NAME="$OPTARG"
-            echo "Specified display name: '$DISPLAY_NAME'" >&2
             ;;
         e)
             ENTITLEMENTS="$OPTARG"
-            echo "Specified signing entitlements: '$ENTITLEMENTS'" >&2
             ;;
         b)
             BUNDLE_IDENTIFIER="$OPTARG"
-            echo "Specified bundle identifier: '$BUNDLE_IDENTIFIER'" >&2
             ;;
         k)
             KEYCHAIN="$OPTARG"
-            echo "Specified Keychain to use: '$KEYCHAIN'" >&2
             ;;
         n)
             VERSION_NUMBER="$OPTARG"
-            echo "Specified version to use: '$VERSION_NUMBER'" >&2
+            ;;
+        v)
+            VERBOSE="--verbose"
             ;;
         \?)
-            echo "Invalid option: -$OPTARG" >&2
-            exit 1
+            error "Invalid option: -$OPTARG"
             ;;
         :)
-            echo "Option -$OPTARG requires an argument." >&2
-            exit 1
+            error "Option -$OPTARG requires an argument."
             ;;
     esac
 done
 
+# Log the options
+for provision in ${RAW_PROVISIONS[@]}; do
+    if [[ "$provision" =~ .+=.+ ]]; then
+        log "Specified provisioning profile: '${provision#*=}' for bundle identifier: '${provision%%=*}'"
+    else
+        log "Specified provisioning profile: '$provision'"
+    fi
+done
+[[ -n "${DISPLAY_NAME}" ]] && log "Specified display name: '$DISPLAY_NAME'"
+[[ -n "${ENTITLEMENTS}" ]] && log "Specified display name: '$ENTITLEMENTS'"
```

Comment:
It looks like lines 150 - 154 include some copy-pasted logging. Should these be as they were above?